SU2_CFD/src/solvers/CNEMOEulerSolver.cpp

```diff
@@ -667,55 +664,59 @@ void CNEMOEulerSolver::SetTime_Step(CGeometry *geometry, CSolver **solver_contai
 
 void CNEMOEulerSolver::SetMax_Eigenvalue(CGeometry *geometry, CConfig *config) {
 
```

Comment:
This function seemed to be causing problems beforehand with the central residual schemes.  I updated it to follow develop.